### PR TITLE
cve-present-line

### PR DESCRIPTION
**Pull request by chekl**

### CODE REVIEW

**Comment by liubomyr123** [comment]
Please, open console in browser and u will see errors:
image

**Comment by liubomyr123** [comment]
Maybe we can use framer motion, try to search some ready components.

**Comment by chekl** [comment]
> Maybe we can use framer motion, try to search some ready components.

Libraries with ready components exist but not for wavy lines like in our design

**Comment by liubomyr123** [comment]
Problem is gone, but line is not long enough for big screens:
[Screencast from 31.08.23 22:15:19.webm](https://github.com/Vladyslav-Samokryk/gift-hub-frontend/assets/62620618/aca5437f-a400-4d0f-b7fa-46fd74c1ff84)

File: src/components/TrickedLine/index.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { useTypedTranslation, useInterval } from "@shared";
+import { useLayoutEffect, useState } from "react";
+import { START_OFFSET, END_OFFSET, MAX_OFFSET, MIN_OFFSET, STEP } from "./constants";
+
+export default function TrickedLine (): JSX.Element {
+  const [path1, setPath1] = useState(START_OFFSET);
+  const [path2, setPath2] = useState(END_OFFSET);
+  const [widthStep, setWidthStep] = useState(0);
+
+  useLayoutEffect(() => {
```
Comment (liubomyr123) [inline src/components/TrickedLine/index.tsx:10]: Why u need here useLayoutEffect?

**Review by liubomyr123 — APPROVED** [review]

File: src/components/TrickedLine/index.tsx (hunk 1 above)
Comment (chekl) [reply src/components/TrickedLine/index.tsx:10]: To meet screen width. If the screen is resizing, this hook will work and the present line will change.

**Pull request merged**